config/core/test-configs.yaml

```diff
@@ -189,6 +189,17 @@ test_plans:
       rootfs_url: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/crostest006/chromiumos-amd64-generic/amd64/'
       fixed_kernel: true
 
+  chromiumosvm-tast-upstream-kernel:
+    pattern: 'chromeos/chromiumosvm-tast-template.jinja2'
+    filters:
+      - blocklist: *kselftest_defconfig_filter
+      - passlist:
+          lab: ['lab-collabora-staging', 'lab-collabora']
+    params:
+      job_timeout: '60'
+      docker_image: 'kernelci/chromeos-tast'
+      rootfs_url: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/crostest006/chromiumos-amd64-generic/amd64/'
```

Comment:
It looks like this wasn't updated using the main rootfs-builder job but one you ran with your branch, as the "pipeline version" is not a date like `20220412.0`.  Maybe that's OK but it's better to be sure it was really built with the `chromeos.kernelci.org` branch so we know it can be reproduced easily.

Reply:
This still hasn't been fixed, the main Chrome OS rootfs build job should have uploaded it to a URL with the current date.